src/pnpq/apt/protocol.py

```diff
@@ -134,6 +134,22 @@ def from_bool(cls, toggle: bool) -> "EnableState":
         return cls.CHANNEL_DISABLED
 
 
+@enum.unique
+class StopMode(int, Enum):
+    """Used in MSMSG_MOT_MOVE_STOP."""
+
+    IMMEDIATE_STOP = 0x01
+    CONTROLLED_STOP = 0x02
+
+
+@enum.unique
+class JogDirection(int, Enum):
+    """Used in MSMSG_MOT_MOVE_JOG."""
+
+    JOG_FORWARD = 0x01
+    JOG_BACKWARD = 0x02
```

Comment:
Removing the redundant "JOG" and using the same words as the specification ("reverse" instead of "backward," not a big deal, but "reverse" is used on cars, too, so it's probably just as good). It's probably best not to merge this suggestion but update it on your machine so you can update any references to it.

```suggestion
   FORWARD = 0x01
   REVERSE = 0x02
```